presto-product-tests-launcher/pom.xml

```diff
@@ -17,6 +17,24 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.prestosql</groupId>
+            <artifactId>presto-testing</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+
+        <!-- used by tests but also needed transitively -->
```

Comment:
This is normally a section at the end of the normal dependencies, before the test dependencies.